--- a/src/meshlab/src/vcglib/wrap/io_trimesh/import_ply.h
+++ b/src/meshlab/src/vcglib/wrap/io_trimesh/import_ply.h
@@ -1089,34 +1089,34 @@ public:
 					if (ReadingOpt.CheckVertexIndexing) curFaceInd++;
 					if (!pf.Read(reinterpret_cast<char *>(&ca), sizeof(camera), &CompatibilityCheck))
 					{
-						this->pi.status = pf.GetError();
-						return this->pi.status;
+						pi.status = pf.GetError();
+						return pi.status;
 					}

 					if (CompatibilityCheck) {
 						// Swap endianness
 					}
 					if (pf.GetError())
 					{
-						return this->pi.status;
+						return pi.status;
 					}

 					// Currently, we read from PlyCamera structure, which is good for binary format.
 					// In ascii format, we must provide custom properties
 					{
 						if (camera_found) {
 							// Overwrite the camera definition that could be already
 							// set from custom properties
-							this->pi.status = PlyInfo::E_SHORTFILE;
-							return this->pi.status;
+							pi.status = PlyInfo::E_SHORTFILE;
+							return pi.status;
 						}
-						this->camera.valid     = true;
-						this->camera.view_p[0] = ca.view_px;
-						this->camera.view_p[1] = ca.view_py;
-						this->camera.view_p[2] = ca.view_pz;
-						this->camera.x_axis[0] = ca.x_axisx;
-						this->camera.x_axis[1] = ca.x_axisy;
-						this->camera.x_axis[2] = ca.x_axisz;
-						this->camera.y_axis[0] = ca.y_axisx;
-						this->camera.y_axis[1] = ca.y_axisy;
-						this->camera.y_axis[2] = ca.y_axisz;
-						this->camera.z_axis[0] = ca.z_axisx;
-						this->camera.z_axis[1] = ca.z_axisy;
-						this->camera.z_axis[2] = ca.z_axisz;
-						this->camera.f         = ca.focal;
+						camera.valid     = true;
+						camera.view_p[0] = ca.view_px;
+						camera.view_p[1] = ca.view_py;
+						camera.view_p[2] = ca.view_pz;
+						camera.x_axis[0] = ca.x_axisx;
+						camera.x_axis[1] = ca.x_axisy;
+						camera.x_axis[2] = ca.x_axisz;
+						camera.y_axis[0] = ca.y_axisx;
+						camera.y_axis[1] = ca.y_axisy;
+						camera.y_axis[2] = ca.y_axisz;
+						camera.z_axis[0] = ca.z_axisx;
+						camera.z_axis[1] = ca.z_axisy;
+						camera.z_axis[2] = ca.z_axisz;
+						camera.f         = ca.focal;
